src/main/java/duke/Storage.java

```diff
@@ -12,6 +12,9 @@
 import java.util.ListIterator;
 import java.util.Scanner;
 
+/**
+ * Handles loading and saving to-do lists to and from the system.
+ */

```

Comment:
Just my personal opinion but I think it would be better if you **standardize all classes' descriptions** as ```Duke.java``` and ```Storage.java``` begin with a _verb_ (```Represents``` and ```Handles```) but for ```DukeException.java``` it starts with a _noun_ (```A representation```)